package.json

```diff
@@ -60,10 +63,8 @@
     ]
   },
   "generator": {
+    "renderer": "react",
     "generator": ">=0.50.0 <2.0.0",
```

Comment:
Lets remember to update this to new new react generator version once that is merged.